### ISSUE TITLE
Staff Registration (I'm not coming button is gone)

### ISSUE THREAD

**Issue by bds002** [magfest/magclassic#48]
For MAGLabs, when our staffers use their registration verification link to complete their registration or to state "They aren't coming" There are suppose to be two buttons at the bottom of the page.

1. To confirm their registration.
2. The other to state they aren't coming to the event.

It would appear with all the recent registration changes that the I'm not coming button has disappeared, Staff have no way of telling the system that they aren't coming to an event.

**Comment by jcooter** [comment]
This doesn't appear to have been resolved?  Needs further testing.

**Comment by kitsuta** [comment]
Yeah, it didn't end up getting fixed. I believe staff can uncheck the 'volunteering' box to unset themselves as staff, then tell us they're not coming, but I'm not sure and in any case that's unintuitive.
